### PR TITLE
update evo ToF outliers in main

### PR DESCRIPTION
**Pull request by jamie-54**
## Main Changes

Discard the outliers for the new Evo ToF sensor 

## Tests

This has been tested on quadcopter and works

### CODE REVIEW

File: raspberry/bzzz/sensors/evo_time_of_flight.py [hunk 1]
```diff
@@ -77,11 +77,11 @@ def __get_measurements_in_background_t(self, serial_path, baud):
 
                 # Checking error codes
                 if rng == 65535:  # Sensor measuring above its maximum limit
-                    dec_out = float('inf')
+                    dec_out = -1
```
Comment (alphaville) [inline raspberry/bzzz/sensors/evo_time_of_flight.py:80]: We've changed these to `nan`, right?

Comment (alphaville) [reply raspberry/bzzz/sensors/evo_time_of_flight.py:80]: @jamie-54 bump

Comment (jamie-54) [reply raspberry/bzzz/sensors/evo_time_of_flight.py:80]: this issue is fixed in the `fix/refactoring-main` branch so I'm going to close this PR and issue

**Comment by jamie-54** [comment]
this issue is fixed in the fix/refactoring-main branch so I'm going to close this PR

**Pull request closed without merging**